configs/nvme-storage-configurator/base/configure.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# Enable debugging
+set -eux
+
+if [ -f /host-etc-systemd-dir/nvme-init-done ]; then
+    echo 'NVMe init already done'
+    exit 0
+fi
+
+cp -f /scripts/configuration.sh /host-usr-local-bin/configuration.sh
```

Comment:
why `-f` is needed?

Reply:
I just added this out of habit of forcefully copying the files to host's file system. Please let me know if `-f` needs to go.

Reply:
You need to decide if it's needed or not. a habit is not a reason to keep it :)

Reply:
removed `-f` option